wtb.js

```diff
@@ -8,19 +8,18 @@
   var aspect = "aspect"
   var height = "height"
   var width = "width"
-  var probe = Object.getPrototypeOf
-  var prone = Object.prototype
-  var own = prone.hasOwnProperty
+  var own = {}.hasOwnProperty
 
   function get(o, k) {
-    var ban = probe && prone === probe(o) && !own.call(o, k)
+    var ban = o instanceof Object && !own.call(o, k)
     if (ban) return
     var v = o[k]
     return typeof v == "function" ? v.call(o) : v
   }
 
   function wtb(given) {
-    var num = +given
+    var sob = typeof given == "object" != given instanceof Object
+    var num = sob && given ? NaN : +given
```

Comment:
`sob` is needed because `+Object.create(null)` throws `Cannot convert object to primitive value` error

Reply:
`sob` becomes `true` for
- functions
- `null`
- `Object.create(null)`